ex20_ch4.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constants
+EC_RES_TAX = 0.005
+DUNN_RES_TAX = 0.004
+IL_SALES_TAX = 8 
+
+#step1: get input from user
+order_amount = float(input("Enter the order amount:$"))
+state = input("Enter the state you reside in. Use abbreviations (eg, OH or MA): ")
+county = input("Enter the county you live in: ")
+
+#step2: set up first conditional
+if "WI" in state or "wi" in state: 
```

Comment:
`if "WI" == state or "wi" == state:`

But I'd suggest what we spoke about yesterday to _normalize_ this input by lowercasing it after you get it from the user. Then you only need to check the first condition:

```
 if "wi" == state:`
  ##
```